cmake/GenerateBuiltins.cmake

```diff
@@ -253,6 +253,10 @@ function (generate_target_builtins resultList)
             endforeach()
         endforeach()
     endforeach()
+    if (WASM_ENABLED)
```

Comment:
Other targets are added in `CMakeFile.txt `in the project root dir (search for ISPC_TARGETS) and `cmake/GenerateBuiltins.cmake` has logic to treat them differently if needed. Could you please follow the same pattern?

Reply:
It doesn't play well when I add "web" to the TARGET_OS_LIST_FOR_LL. It starts generating all sse, avx .ll files for  web.

Reply:
I see, let me try play with it and see how we can solve it.
And just for the background of UNIX/WINDOWS 32/64 macros which go to target .ll preprocessing - they were introduced solely to have proper versions of new/delete operator (they map to aligned malloc/free, which are different on UNIX / WINDOWS and sensitive to 32/64 bits). For the rest of the code of target library we don't care about these differences. So adding WEB OS to this list makes perfect sense to me, as memory allocation will be different.

Reply:
I've pushed commit, which implementation that I had in mind, when I commented that. Please have a look and check if it looks good to you.